scripts/plugins/utilities.py

```diff
@@ -348,10 +348,13 @@ def handle_hooks(mode, hooks_folder, source_folder):
         else:
             logger.warning(f"~#~#~#~{umode} HOOK [{hook_script}] FAILED~#~#~#~")
             logger.debug(result.stdout)
+            if BITOPS_fast_fail_mode:
+                sys.exit(result.returncode)
+
     os.chdir(original_directory)
 
 
-def run_cmd(command: Union[list, str]) -> subprocess.CompletedProcess:
+def run_cmd(command: Union[list, str]) -> subprocess.Popen:
     """Run a linux command and return CompletedProcess instance as a result"""
```

Comment:
Also missing a correct `Popen` reference in the function description: 
```suggestion
    """Run a linux command and return subprocess.Popen instance as a result"""
```

outside of that, looks good!

Reply:
Shoot sorry